mxcube3/core/components/queue.py

```diff
@@ -1077,19 +1094,13 @@ def set_dc_params(self, model, entry, task_data, sample_model):
                 sample_model
             )
 
-        full_path = os.path.join(
-            HWR.beamline.session.get_base_image_directory(), params.get("subdir", "")
+        full_path, process_path = HWR.beamline.session.get_full_path(
```

Comment:
get_full_path returns both image and process directory and optionally handles (ESRFSession) run numbers on directory level (one directory per run)